eras/conway/impl/cddl-files/extra.cddl

```diff
@@ -11,9 +15,9 @@ unit_interval = #6.30([1, 2])
   ; but this produces numbers outside the unit interval
   ; and can also produce a zero in the denominator
 
-rational =  #6.30([uint, positive_int])
+rational =  #6.30([int, positive_int])
 
-positive_interval = #6.30([uint, positive_int])
+nonnegative_interval = #6.30([uint, positive_int])
```

Comment:
Are we replacing positive_interval with nonnegative_interval, or is the intent to support both
positive_interval and nonnegative interval?

Reply:
We have no uses of `positive_interval` anywhere in our types, so that's why I replaced it with `nonnegative_interval`